### ISSUE TITLE
/usr/bin/id: cannot find name for group ID 10513

### ISSUE THREAD

**Issue by flamecopper** [babun/babun#612]
I have first installed babun. When I tried to launch I got this error message

**Comment by flamecopper** [comment]
Yes. I have Cygwin installed as well. I have use it for a year. Babun is new for me.

**Comment by chris-maier** [comment]
Had the same issue. The following link fixed my problem. 

[link](http://serverfault.com/questions/666639/cygwin-startup-error-usr-bin-id-cannot-find-name-for-group-id)

**Comment by Sylk** [comment]
Please close this.
